app/assets/stylesheets/_main.scss

```diff
@@ -42,18 +42,22 @@ $govuk-page-width: 1100px;
 
 p {
   @extend .govuk-body;
+  color: inherit;
 }
 
 h1 {
   @extend .govuk-heading-l;
+  color: inherit;
```

Comment:
Mixing in the GOV.UK styles to element selectors was forcing the text colour to be blank so reset `color` back to `inherit` to avoid this rather than trying to add `color` definitions to all the other affected styles.